include/net/ieee802154_radio.h

```diff
@@ -30,6 +30,7 @@ enum ieee802154_hw_caps {
 	IEEE802154_HW_FILTER		= BIT(2),
```

Comment:
Something does not sound right in commit message, could it be written something like this instead:

```
Sub-Ghz bands can have different limit of channels, typically 10 or more than thousand are possible.
This means that the device needs to expose such limit to the L2 which is unaware of frequency band logic. L2 will then allow user to select a proper channel.
```

Reply:
done